docs/how-to/test-your-deployment.md

```diff
@@ -0,0 +1,42 @@
+# How to test your deployment
+
+In this document, we describe the main tests to ensure your deployment is working properly.
+
+All actions should be performed after your deployment is successful (all units and apps in `idle` state).
+
+## Test the dashboard
+
+First retrieve valid credentials:
+
+- Go to the environment where your `wazuh-indexer` is deployed.
+- Run `juju run data-integrator/leader get-credentials` and write down the `username` and `password`.
+
+Then access the dashboard:
+
+- Go to the environment where your `wazuh-dashboard` is deployed.
+- Retrieve the leader IP address.
+- Connect from your browser to `https://$leader_ip:5601`.
+- Accept the security exception.
```

Comment:
I don't think we should mention this. The certificate might be valid depending on the set up